tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -192,6 +192,12 @@ def create_parser(product_choices=None):
                               help="Path to directory containing extra json files to add to run info")
     config_group.add_argument("--product", action="store", choices=product_choices,
                               default=None, help="Browser against which to run tests")
+    config_group.add_argument("--browser-version", action="store",
+                              default=None, help="Informative string detailing the browser "
+                              "release version. This is included in the run_info data.")
```

Comment:
In my earlier change, I added this to `wpt/run.py`:

https://github.com/web-platform-tests/wpt/blob/1de1550e95396f979da97620a289c213acaf55e2/tools/wpt/run.py#L439

Would it always override the command line argument?